src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -258,6 +261,7 @@
 def load_arguments(self, _):
 
     acr_arg_type = CLIArgumentType(metavar='ACR_NAME_OR_RESOURCE_ID')
+    k8s_support_plans = self.get_models("KubernetesSupportPlan", resource_type=ResourceType.MGMT_CONTAINERSERVICE, operation_group='managed_clusters')
```

Comment:
```suggestion
    k8s_support_plans = self.get_models("KubernetesSupportPlan", resource_type=ResourceType.CUSTOM_MGMT_AKS_PREVIEW, operation_group='managed_clusters')
```

In aks-preview, should take the data model from preview API.